packages/server/src/ui/routes/build-view/audit-detail/audit-detail-pane.stories.jsx

```diff
@@ -53,6 +57,7 @@ const auditPairs62 = createAuditPairs(lhr62A, lhr62B);
 const auditPairs641 = createAuditPairs(lhr641A, lhr641B);
 const auditPairs700 = createAuditPairs(lhr700A, lhr700B);
 const auditPairs800 = createAuditPairs(lhr800A, lhr800B);
+const auditPairs930 = createAuditPairs(lhr930A, lhr930B);
```

Comment:
![image](https://user-images.githubusercontent.com/4071474/151904438-217e1741-edc1-41a1-8b1c-7a9de99cc47b.png)

https://github.com/GoogleChrome/lighthouse-ci/blob/main/packages/server/src/ui/routes/build-view/audit-detail/audit-detail-pane.stories.jsx#L166

Not sure why the audit pairs for the other versions don't have this problem, but for the 9.3.0 LHR I'm getting a diff at `screenshot-thumbnails`, as you'd expect. But the viz is non existent so we just print the JSON.

Not sure what the best approach to deal with this is. Is it problematic that this audit is presented this way? How would we prevent it from ever being displayed? Should I just skip it in the `createAuditPairs` like `uses-long-cache-ttl` is skipped?

Reply:
confirmed this is existing behavior in https://googlechrome.github.io/lighthouse-ci/viewer/